src/index.css

```diff
@@ -1,7 +1,7 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: Lato-Regular, sans-serif;
```

Comment:
just double checking the font is being imported somewhere?

Reply:
Good question! i'll check

Reply:
It is...removed this rule

Reply:
still need it set :)